eventuate-tram-spring-reactive-commands-starter/src/main/java/io/eventuate/tram/spring/reactive/commands/autoconfigure/EventuateReactiveTramCommandsAutoConfigure.java

```diff
@@ -0,0 +1,13 @@
+package io.eventuate.tram.spring.reactive.commands.autoconfigure;
+
+import io.eventuate.tram.spring.reactive.commands.consumer.ReactiveTramCommandConsumerConfiguration;
+import io.eventuate.tram.spring.reactive.commands.producer.ReactiveTramCommandProducerConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@ConditionalOnClass(ReactiveTramCommandConsumerConfiguration.class)
```

Comment:
Is it possible for `EventuateReactiveTramCommandsAutoConfigure` to be on the classpath but not `ReactiveTramCommandConsumerConfiguration`?

Reply:
It seems no, I will remove

Reply:
updated